src/server/pfs/server/server_test.go

```diff
@@ -1704,22 +1723,6 @@ func TestPutFileWithNoDelimiter(t *testing.T) {
 
 }
 
-func TestPutFileNullCharacter(t *testing.T) {
```

Comment:
Are we just removing this because sha256 doesn't like null characters? Are null characters not supported anymore?